site/_data/contributors.yml

```diff
@@ -142,6 +142,11 @@
   githubId: jcamachor
   org: Hortonworks
   role: PMC
+- name: Jiajun Xie
```

Comment:
Would you mind to also add `pronouns` section like https://github.com/apache/calcite/blob/0dcf3ab4185647243255af29f0128b52737c1bf2/site/_data/contributors.yml#L51 ?

Reply:
Thanks for your review.

Most people didn't add `pronouns`, I prefer to be the same as most people.